it/mail/chrome/messenger/messengercompose/messengercompose.dtd

```diff
@@ -249,7 +249,7 @@
 <!ENTITY SmileButton.tooltip "Inserisci una faccina">
 <!ENTITY smiley1Cmd.label "Sorriso">
 <!ENTITY smiley2Cmd.label "Broncio">
-<!ENTITY smiley3Cmd.label "Ammicco">
+<!ENTITY smiley3Cmd.label "Ammico">
```

Comment:
CORRECTIONS NEEDED. The sentence in red was correct. Please revert to: 

+<!ENTITY smiley3Cmd.label "Ammicco">